person.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'rental'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
```

Comment:
- Hey, when creating a rental, it throws an error as you've not added the book to the attr_accessor. this should be updated to `  attr_accessor :name, :age, :books, :rentals` to fix this error

![image](https://github.com/Koshao-sys/OOP-school-library/assets/59397159/bbabb133-9940-4c71-8787-d8788ff098cb)